builder/scaleway/step_create_server.go

```diff
@@ -28,18 +28,21 @@ func (s *stepCreateServer) Run(_ context.Context, state multistep.StateBag) mult
 	}
 
 	if c.Comm.SSHPublicKey != nil {
-		tags = []string{fmt.Sprintf("AUTHORIZED_KEY=%s", strings.Replace(strings.TrimSpace(string(c.Comm.SSHPublicKey)), " ", "_", -1))}
+		env = fmt.Sprintf("AUTHORIZED_KEY=%s", strings.Replace(strings.TrimSpace(string(c.Comm.SSHPublicKey)), " ", "_", -1))
```

Comment:
This change doesn't seem directly related to allowing empty images; can you explain it a bit?